app/views/admin/npq/applications/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<% content_for :title, t(".title") %>
+
+<%= render "admin/npq/applications/layout", locals = { page: "applications" } %>
+
+<div class="admin-search-box">
+      <%= form_with method: :get do |f| %>
+        <%= f.govuk_text_field(
+            :query,
+            value: params[:query],
+            label: { text: "Search records", size: "s" },
+          )
+        %>
+
+        <%= f.govuk_submit "Search" %>
+      <% end %>
+    </div>
+
+<table class="govuk-table">
```

Comment:
Worth using [the table component](https://govuk-components.netlify.app/components/table/) here?

Reply:
Yeah this sounds better i can look into this